update.sh

```diff
@@ -43,19 +57,32 @@ for version in "${versions[@]}"; do
 	md5="${fullVersion##* }"
 	fullVersion="${fullVersion% $md5}"
 
+	if [ "$version" != '7' ]; then
+		md5='composer'
+	fi
 	echo "$version: $fullVersion ($md5)"
```

Comment:
Since this is just for display, we might as well just remove it instead:

```suggestion
	echo "$version: $fullVersion"
```